website/docs/r/ebs_encryption_by_default.html.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: "aws"
+page_title: "AWS: aws_ebs_encryption_by_default"
+sidebar_current: "docs-aws-ebs-encryption-by-default"
+description: |-
+  Manages whether default EBS encryption is enabled for your AWS account in the current AWS region.
+---
+
+# Resource: aws_ebs_encryption_by_default
+
+Provides a resource to manage whether default EBS encryption is enabled for your AWS account in the current AWS region.
```

Comment:
Might be worth a cross link to the `aws_ebs_default_kms_key` resource here as well, e.g.

```suggestion
Provides a resource to manage whether default EBS encryption is enabled for your AWS account in the current AWS region. To manage the default KMS key for the region, see the [`aws_ebs_default_kms_key` resource](/docs/providers/aws/r/ebs_default_kms_key.html).
```